src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable no-nested-ternary */
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { PeopleFilters } from '../PeopleFilters';
+
+export const PeoplePage = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(true);
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+  const query = searchParams.get('query');
+  const sex = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries');
+
+  useEffect(() => {
+    getPeople()
+      .then((data) => setPeopleFromServer(data))
+      .catch(() => setError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  let visiblePeople = [...peopleFromServer];
```

Comment:
There should be a separate function from line 29 to 75

Reply:
OK, gonna fix it, thank you!